index.js

```diff
@@ -10,7 +10,8 @@ const correctMessage = document.getElementById('correct');
 
 let targetNumber;
 let attempts = 0;
-const maxNumberOfAttempts = 5;
+//If we want to change we must use let
+let maxNumberOfAttempts = 5;
```

Comment:
Currently, game resetting doesn't work. When you reset the game, your attempts start to become negative. That's because the code tries changing `maxNumberOfAttempts` instead of resetting `attempts` on line 86.

So instead of using `let` for this variable and changing that to reset it to 5. You need to just reset the `attempts` variable.

Reply:
Do you mean that I should change to 0  let maxNumberOfAttempts = 0;
and then in the line 86 just leave maxNumberOfAttempts = 5;